scripts/in_container/prod/entrypoint_prod.sh

```diff
@@ -311,6 +311,10 @@ if [[ -n "${_AIRFLOW_WWW_USER_CREATE=}" ]] ; then
     create_www_user
 fi
 
+if [[ -n "${_PIP_ADDITIONAL_REQUIREMENTS=}" ]] ; then
+    pip install --no-cache-dir --user "${_PIP_ADDITIONAL_REQUIREMENTS=}"
```

Comment:
I wonder if it is worth adding a warning here that this option has a drastic impact on performance and recommend migrating to the standard method. It will be most painful if the user is to use the `airflow.sh` script as it has been optimized for speed and this option will have a significant impact on it.

Reply:
Added

Reply:
```
        >&2 echo "The container installs new packages at startup. For a better performance/less network overhead , consider bake dependencies to image."
```